Tasks/Korotkevich_Grigory/Task_2/Task2.py

```diff
@@ -0,0 +1,98 @@
+
+'''Реализовать:
+1.текстовый вывод сегодняшнего времени
+2.текстовый вывод времени, ведомого с консоли (чч:мм).
+(дать возможность выбора режима работы программы)
+
+Для получения свободного времени можно использовать модуль datetime.
```

Comment:
google translate and back? very strange wording, not from my task description

Reply:
Yes, it is he who is the culprit of the distortion :)